composer.json

```diff
@@ -13,28 +16,28 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "roave/security-advisories": "dev-master",
+        "zendframework/zend-component-installer": "^0.7.0",
```

Comment:
We've had BC breaks in the API between versions, but not in *how* it's consumed. I think in this particular case, consumption is the key factor, so we can re-add the `^1.0` constraint. I'll do that on merge.